styles/SASS/_config.scss

```diff
@@ -1,13 +1,35 @@
 // Placeholder for variables, colors, mixins etc.
 
 // Example of variable
-$background-color: white;
-$text-color: black;
-$mobile: 800px;
+$color-red: #9b1d20ff;
+$color-black: #020402ff;
+$color-white: #e8f1f2ff;
+$color-blue: #235789ff;
+$color-yellow: #e3b505ff;
```

Comment:
maybe rename it with some names like "color-primary", "color-text" etc. Other devs might don't know when to use which color.
Or maybe add alias for it? 🤷

Reply:
I think that the way Gosia made it is better, because we may mix these colors and not always use them for the same purpose.

Reply:
we got two text-colors (black and white depends on the background-color) and three primary colors: red for the start view and blue or yellow for the fight view (depends on the current player)

Reply:
It is good idea to add "color-primary" and "color-text" names.  It could help in code reading.